.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "protos"]
 	path = protos
-	url = https://github.com/fishjam-cloud/protos.git
+	url = git@github.com:fishjam-cloud/protos.git
```

Comment:
Are you sure we wont this change? Our other repos use `https` in url

Reply:
as discussed, let's leave it with ssh version.